internal/indexer/data/redis_client.py

```
@@ -0,0 +1,149 @@
+import time
+import redis
+import logging
+from utils.constants import *
```

Ruff (0.15.9)

[error] `from utils.constants import *` used; unable to detect undefined names

(F403)